docker-images/pysyft-worker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6-slim
+
+ENV WORKER_DIR /worker
+
+# Install PySyft
+RUN apt-get update && apt-get install -y gcc
+RUN pip install --no-cache-dir syft
```

Comment:
This layer will be cached with whatever syft version is available at the time of the first build and won't change until you reset docker build cache.
It is better to specify syft version explicitly via ARG or ENV

Reply:
I'm doing it this way cause I think of automating the build process, and whenever you pull the image from the docker hub you have that last stable version available on pip, adding the version here will add an extra management, to keep the Dockerfile updated, right?

Reply:
Then you'll probably have to disable caching :). Oh, and somehow make sure the latest syft is available for pip before docker build is triggered?

Reply:
I'm not yet sure about the tool, but I think that the build process is generally made on a fresh instance and the image shouldn't be cached their, I will just make sure to setup the trigger correctly like you said